molgenis-web/src/main/java/org/molgenis/web/exception/BadRequestController.java

```diff
@@ -0,0 +1,24 @@
+package org.molgenis.web.exception;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import static org.molgenis.web.exception.BadRequestController.URI;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+@Controller
+@RequestMapping(value = URI)
+public class BadRequestController
+{
+	public static final String ERROR_MESSAGE = "The page you’re looking for can't be found.";
```

Comment:
add (reintroduce) notFoundController with this message, update this message to a bad request message